front/components/assistant/conversation/UserMessage.tsx

```diff
@@ -34,7 +34,7 @@ export function UserMessage({
       user={user}
       conversationId={conversation.sId}
       messageId={message.sId}
-      pictureUrl={message.context.profilePictureUrl}
+      pictureUrl={message.user?.image || message.context.profilePictureUrl}
```

Comment:
why not `user.imageUrl || message.context.profilePictureUrl` ?

Reply:
User here is the currently logged in user.

Reply:
Oooops indeed 😄